src/ui/common/src/components/workflows/StatusBar.tsx

```diff
@@ -192,14 +198,16 @@ const ActiveWorkflowStatusTab: React.FC<ActiveWorkflowStatusTabProps> = ({
               >
                 {listItem.title}
               </Typography>
+
               <Typography
                 sx={{
                   fontFamily: 'Monospace',
                   fontWeight: 'light',
                   marginTop: '2px',
                   fontSize: '12px',
-                  whiteSpace: 'pre-wrap',
+                  textOverflow: 'wrap',
```

Comment:
what does this change?

Reply:
I was trying to get the text to wrap, but I was unable to.